packages/orbit-components/src/HorizontalScroll/HorizontalScroll.stories.tsx

```diff
@@ -96,86 +84,147 @@ export const NoScroll = ({ spacing }) => {
               </div>
             </Text>
           </Box>
-        ))}
-      </HorizontalScroll>
-    </Stack>
-  );
-};
-
-NoScroll.args = {
-  spacing: SPACINGS.FIFTY,
-};
+        ))
+      }
+    </HorizontalScroll>
+  ),
 
-NoScroll.argTypes = {
-  spacing: {
-    options: Object.values(SPACINGS),
-    control: {
-      type: "select",
+  parameters: {
+    controls: {
+      disable: true,
     },
   },
+
+  args: {
+    spacing: undefined,
+    scrollPadding: undefined,
+    scrollSnap: undefined,
+    overflowElevation: false,
+  },
 };
 
-export const WithScrollSnap = ({
-  scrollSnap,
-  scrollPadding,
-  overflowElevation,
-  elevationColor,
-  scrollSnapAlign,
-}) => {
-  return (
-    <HorizontalScroll
-      scrollSnap={scrollSnap}
-      scrollPadding={scrollPadding}
-      overflowElevation={overflowElevation}
-      elevationColor={elevationColor}
-    >
-      {Array(...Array(10)).map((_, key) => (
-        <div
-          style={{
-            scrollSnapAlign,
-            width: "150px",
-            display: "flex",
-            justifyContent: "center",
-            height: "50px",
-            background: "#ccc",
-          }}
-        >
-          <Text size="large" weight="bold" as="div">
-            <div
-              style={{
-                height: "50px",
-                display: "flex",
-                alignItems: "center",
-              }}
+export const NoScroll = {
```

Comment:
What does NoScroll mean in this context?

I can scroll it on small devices. What am I missing?

Reply:
It's a "garbage" from the old storybook.

I am considering to remove "with scroll snap" as well and keep only `Default` and `Playground` stories, because "With scroll snap" and "Playground" will be the same and this component is "simple" in general. 🤔

Reply:
Not sure if "garbage" is the best definition. I'm sure this story was added for a specific purpose, I was just wondering what it was.

Reply:
yeah, maybe "garbage" is too strong. 🙈  in the [old storybook](https://kiwicom-orbit-sarka-storybook-migration-b4.surge.sh/?path=/story/horizontalscroll--no-scroll ) there are only a few items in the component and it demonstrates that scrolling is available only when the width of all items is bigger than parent contaner's width.